corehq/apps/userreports/expressions/getters.py

```diff
@@ -75,6 +76,10 @@ def transform_date(item):
     return item or None
 
 
+def transform_datetime(item):
+    return item or None
```

Comment:
Following the example of `transform_date(item)` here, but it seems like this could be doing some transforming? My guess is that either the database or sqlalchemy converts strings to datetimes at insert time, but it might not hurt to do that conversion explicitly here.